### PR TITLE
merge from develop

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

File: src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+          IsActive = table.Column<bool>(nullable: false),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+          LeftAtUtc = table.Column<DateTime>(nullable: true)
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PK_DepartmentUser", x => x.Id);
```
Comment (asyagraf) [inline src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs:57]: incorrect name for PrimaryKey (PR_DepartmentsUsers)

File: src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          DepartmentId = table.Column<Guid>(nullable: false),
+          IsActive = table.Column<bool>(nullable: false),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PK_DepartmentNews", x => x.Id);
```
Comment (asyagraf) [inline src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs:78]: same (PK_DepartmentsNews)

File: src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          DepartmentId = table.Column<Guid>(nullable: false),
+          IsActive = table.Column<bool>(nullable: false),
+          CreatedBy = table.Column<Guid>(nullable: false),
+          CreatedAtUtc = table.Column<DateTime>(nullable: false),
+          ModifiedBy = table.Column<Guid>(nullable: true),
+          ModifiedAtUtc = table.Column<DateTime>(nullable: true),
+        },
+        constraints: table =>
+        {
+          table.PrimaryKey("PK_DepartmentProject", x => x.Id);
```
Comment (asyagraf) [inline src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs:99]: same (PK_DepartmentsProjects)

**Review by asyagraf — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs (hunk 1 above)
Comment (MariyaSin) [reply src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs:57]: we will fix it in next task

File: src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs (hunk 2 above)
Comment (MariyaSin) [reply src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs:78]: same

File: src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs (hunk 3 above)
Comment (MariyaSin) [reply src/DepartmentService.Data.Provider.MsSql.Ef/Migrations/20211012014055_InitialCreate.cs:99]: same

**Pull request merged**